src/PowerShellEditorServices/Session/Host/EditorServicesPSHostUserInterface.cs

```diff
@@ -684,6 +688,60 @@ private void WriteDebuggerBanner(DebuggerStopEventArgs eventArgs)
             }
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        public static ConsoleColor BackgroundColor
+        {
+            get;
+            set;
+        }
```

Comment:
This should follow the same style as below (e.g. `public static ConsoleColor BackgroundColor { get; set; }`)